packages/subgraph/schema.graphql

```diff
@@ -391,7 +412,7 @@ type AdminProposal implements Proposal @entity {
 type MultisigPlugin implements IPlugin @entity {
   "TODO: attributes should be appended to Plugin once plugin is seperated from core"
   id: ID!
-  daos: [DaoPlugin!] @derivedFrom(field: "plugin")
+  # daos: [DaoPlugin!] @derivedFrom(field: "plugin")
```

Comment:
how we know on wich DAO this plugin is installed?
are you planing to do it in the next PR?